chapters/en/chapter6/evaluation.mdx

```diff
@@ -0,0 +1,28 @@
+# Evaluating text-to-speech models
+
+During the training time, text-to-speech models optimize for the mean-square error loss (or mean absolute error) between 
+the predicted spectrogram values and the generated ones. Both MSE and MAE encourage the model to minimize the difference 
+between the predicted and target spectrograms. However, evaluation of the resulting text-to-speech (TTS) models is much 
```

Comment:
```suggestion
between the predicted and target spectrograms. However, since TTS is a one-to-many mapping problem, i.e. the output spectrogram for a given text can be represented in many different ways, the evaluation of the resulting text-to-speech (TTS) models is much 
```

Just so we clarify that the main problem here is not MSE but the fact that the output spectrogram can have multiple legal representations.